src/Apps/Order2/Routes/Checkout/Components/Order2ReviewStep.tsx

```diff
@@ -27,14 +28,16 @@ export const Order2ReviewStep: React.FC<Order2ReviewStepProps> = ({
   const artwork = orderData.lineItems[0]?.artwork
   const artworkVersion = orderData.lineItems[0]?.artworkVersion
   const submitOrderMutation = useSubmitOrderMutation()
-  const { steps, confirmationToken, redirectToOrderDetails } =
+  const { steps, confirmationToken, redirectToOrderDetails, checkoutTracking } =
     useCheckoutContext()
+
   const stepState = steps?.find(
     step => step.name === CheckoutStepName.CONFIRMATION,
   )?.state
 
   const [loading, setLoading] = useState(false)
-  const handleClick = async event => {
+  const handleClick = async _event => {
+    checkoutTracking.submittedOrder()
```

Comment:
Question: Should we move the tracking call to when the submission actually happens? just to prevent tracking in case the mutation fails